usaspending_api/conftest_helpers.py

```diff
@@ -103,16 +104,28 @@ def __init__(self, index_type):
             "write_alias": self.index_name + "-load-alias",
             "process_deletes": True,
         }
+
+        match self.index_type:
+            case "award":
+                es_id_field = "award_id"
+                primary_key_field = "award_id"
+            case "subaward":
+                es_id_field = "broker_subaward_id"
+                primary_key_field = "broker_subaward_id"
+            case _:
```

Comment:
Might be worth matching on "transaction" and let the default case raise an error to help avoid any future indexes overlooking this.

Reply:
Added a case for `transaction` and raised an error in all other cases.